department-of-veterans-affairs/caseflow
# No "Mark complete" action for SendCavcRemandProcessedLetterTask that is assigned to an org

Resolves #15898

### Description
Remove the "Mark complete" action for SendCavcRemandProcessedLetterTask assigned to an org

### Acceptance Criteria
- [ ] No "Mark complete" action for `SendCavcRemandProcessedLetterTask` that is assigned to an org
- [ ] "Mark complete" action for `SendCavcRemandProcessedLetterTask` that is assigned to a user

### Testing Plan

1. Login as a CAVC lit support user
2. Go to http://localhost:3000/organizations/cavc-lit-support?tab=unassignedTab
3. Select an appeal with "Send CAVC-Remand-Processed Letter Task" to view the Case Details page. Note the docket number and UUID so you can find this appeal later.
4. Note that "Mark as complete" is not an available task action. Select "Assign to person". Select some user, possibly yourself. If you choose another user, log in as that other user.
5. Go to Case Details for the case you just modified.
7. Check the task tree.
6. Select "Mark as complete" and submit.
7. Check the task tree.  Play around and stress test.


### User Facing Changes
 - [ ] Screenshots of UI changes

 BEFORE|AFTER
 ---|---
![image](https://user-images.githubusercontent.com/55255674/109550684-3aca8a00-7a95-11eb-99f6-455602ba6c50.png) | ![image](https://user-images.githubusercontent.com/55255674/109556100-fa223f00-7a9b-11eb-8edb-680d6ef5f09b.png)
